.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+
+name: Test
+
+on: push
+
+env:
+  OTP_VERSION_SPEC: "24.1"
+  ELIXIR_VERSION_SPEC: "1.12.3"
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-20.04
+    services:
+      postgres:
+        env:
+          PGUSER: postgres
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_HOST: localhost
+          POSTGRES_PORT: 5432
+          POSTGRES_HOST_AUTH_METHOD: trust
+        image: postgres:9.6
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready --health-interval 10s
+          --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
```

Comment:
Do you need ruby in this?

Reply:
I did at this point, but not anymore since we're removing coveralls.